ok
2
Traceback (most recent call last):
  File "/home/pcarbonn/Documents/repos/IDP-Z3/test.py", line 60, in generateZ3
    idp.execute()
  File "idp_solver/Run.py", line 79, in execute
    exec(main, mybuiltins, mylocals)
  File "<string>", line 3, in <module>
NameError: name 'u' is not defined

